src/operations/utils/flatten.js

```diff
@@ -0,0 +1,8 @@
+// flatten the given argument into a single flat array
```

Comment:
likewise for this one, this is an array helper that belongs with other array helpers, not in operations

Reply:
i don't think there will be a core after this reorganization, but there could be some common utility functions. Should i move this to a 'utils' directory outside of operations?

fyi, there are utils in the math directory as well, but specific to the math functionality.

Reply:
hmm good point, arrayUtils ? I honestly use the array helpers in every single design, and they are used all over in whole UI, CLI, and all other parts as well, so might as well have it in a reusable place

Reply:
Good point. Let’s move those to a more useful area for users. By the way, it also means documentation and tests need to be written.

Reply:
this has been moved to src/utils/flatten.js

not perfect because the CSG library still lives outside of the BIG repository but at least a good starting point for common utility functions.

i will add some documentation and tests if this is a good compromise.

thoughts?